packages/contracts/contracts/Passport.sol

```diff
@@ -187,4 +194,43 @@ contract Passport is ERC721, Ownable, AccessControlEnumerable {
       return minters;
   }
 
+  /**
+    * @dev Airdrop tokens to a list of address. Restricted to admins.
+    * @param accounts The members to which token need to be airdropped.
+    */
+  function airdrop(address[] memory accounts)
+      external
+      payable
+      onlyOwner
+  {
+      require(supply >= accounts.length, "Error, number of accounts exceeds supply");
+      require(msg.value >= price*accounts.length, "Error, Airdrop costs more");
+      require(!isPrivate || isMinter(msg.sender), "Address is not allowed to mint");
```

Comment:
the `onlyOwner` modifier makes this redundant?

Reply:
We donot add owner to list of minters by default